.github/workflows/c-cpp.yml

```diff
@@ -80,6 +86,15 @@ jobs:
         sudo apt-get install gcc-multilib g++-multilib # bits/libc-header-start.h, bits/c++config.h
     - name: Build
       run: make ARCH=x86 -f Makefile.RAHasher
+    - name: Zip
+      if: startsWith(github.ref, 'refs/tags/')
+      run: make ARCH=x86 -f Makefile.RAHasher zip
+    - name: Release
+      uses: softprops/action-gh-release@v1
+      if: startsWith(github.ref, 'refs/tags/')
```

Comment:
These `if: startsWith(github.ref, 'refs/tags/')` make sure that these steps will only execute when what triggered this workflow was a tag. Commits to a typical branch will not trigger these steps.